app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -36,6 +37,9 @@ public TranslationsSpinnerAdapter(Context context,
     // and the String[] constructor makes a new (immutable) List with the items of the array.
     super(context, resource, new ArrayList<>());
     this.layoutInflater = LayoutInflater.from(context);
+    List<String> transList = new ArrayList<String>(Arrays.asList(translationNames));
```

Comment:
ah, makes sense, because `Arrays.asList` generates an immutable list.

i'd then replace it with a loop - i.e.

```java
List<String> transList = new ArrayList<>();
for (String translation : translationNames) {
   transList.add(translation);
}
```